corehq/apps/consumer_user/urls.py

```diff
@@ -0,0 +1,24 @@
+from django.urls import path
+
+from .views import (
+    change_contact_details_view,
+    change_password_view,
+    domains_and_cases_list_view,
+    login_view,
+    logout_view,
+    register_view,
+    success_view,
+)
+
+app_name = 'consumer_user'
+
+urlpatterns = [
+    path('signup/<invitation>/', register_view, name='consumer_user_register'),
```

Comment:
Thoughts on changing `invitation` to something like `invitation_code`?  I'd expect `invitation` to be an instance of `ConsumerUserInvitation`, but in this PR it sometimes refers to an object instance, and sometimes to the timestamped primary key.  It'd be nice to be consistent about that.